### DISCUSSION TITLE
Unable to register user on first hit

### DISCUSSION THREAD

**Discussion by shaqb** [ludeknovy/jtl-reporter#215] [General]
Hi team,

I have followed the instructions to install the jtlreporter on my MacBook. I have pulled and run the docker compose and everything starts cleanly.

However when I get the initial login screen and add my username and password for the first time I get an error:

![image](https://user-images.githubusercontent.com/19668009/236427912-df30c1a0-f97e-4a3d-8ae3-4bc5462b195d.png)

And looking at the docker logs for the jtl-reporter-be-1 I get the following error:

`{"code":"42703","file":"parse_target.c","length":122,"level":"error","line":"1034","name":"error","position":"43","routine":"checkInsertTargets","severity":"ERROR"}
{"level":"error","message":"Unexpected error: error: column \"role\" of relation \"users\" does not exist, errorId: ea8fc656-d9c5-47a3-83ff-14fb76dae5d0"}
{"level":"info","message":"POST /api/auth/initialize-user 500 72ms","meta":{}}`

I've tried this on v4.8.3 and v4.8.2 and the issue is the same on both. Any ideas?

**Comment by shaqb** [comment]
Actually restarting the compose fixed it!

**Reply by ludeknovy** [reply]
Hi! The thing is, it takes some time to finish the database migrations on the first run. Once it's done, it works, and the error above is simply saying that it was not completed (a column missing on a table).

**Reply by shaqb** [reply]
Thanks @ludeknovy - that makes sense
